frontend/pages/approve.tsx

```diff
@@ -11,10 +11,32 @@ import { capitalizeFirstLetter } from 'utils/capitalizeFirstLetter'
 import { useStakeConnection } from 'hooks/useStakeConnection'
 import { useSplitRequest } from 'hooks/useSplitRequest'
 
+async function getStakeAccountsPubkeys(user: PublicKey, stakeConnection : StakeConnection){
```

Comment:
Putting this here instead of StakeConnection because the frontend no longer uses the monorepo version of `@pythnetwork/staking`